src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const books = createSlice({
+  name: '',
+  initialState: [],
+  reducers: {
+    addBook: (state) => state,
+    removeBook: (state) => state,
```

Comment:
- [ ] The logic for adding and removing book(s) is missing from your implementation, kindly reference this [tutorial](https://medium.com/@ralph1786/adding-and-removing-items-from-redux-store-6d1303ed32c6). ℹ️ Every time object is added or remove via the reducer, twoactions are performed to update the array either to add or remove. 🙏 

<img width="643" alt="Untitled" src="https://github.com/sheezu900/Bookstore/assets/103386183/c43f9645-3507-4cc4-8036-a9093959f90e">